main.js

```diff
@@ -227,10 +249,12 @@ function renderApp(app) {
   var $view = document.querySelector('[data-view="' + app.view + '"]')
   if (app.view === 'catalog') {
     $view.innerHTML = ''
+    $view.appendChild(renderCartCount(app.cart))
     $view.appendChild(renderCatalog(app.catalog.items))
   }
-  if (app.view === 'details') {
+  if (app.view === 'details' || app.view === 'cart') {
```

Comment:
The cart view is not until a later feature (view cart summary). The cart count should be visible at all times.